app/index.tsx

```diff
@@ -11,6 +11,7 @@ import { classifierStateType } from './reducers/types';
 import { statusOnline } from './actions/online';
 import { logError } from './actions/logError';
 import './app.global.css';
+import './styles/app.global.scss';
```

Comment:
This is the only place where I would expect the `app.global.scss` to be imported to :wink: